images/Dockerfile.ubuntu

```diff
@@ -143,73 +148,73 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     polkitd \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-FROM base-ubuntu-22-lts AS hwe-22-lts
+FROM base-ubuntu-22.04 AS hwe-22.04
 RUN apt-get update && apt-get install -y --no-install-recommends \
     linux-image-generic-hwe-22.04 \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-FROM base-ubuntu-20-lts AS hwe-20-lts
+FROM base-ubuntu-20.04 AS hwe-20.04
 RUN apt-get update && apt-get install -y --no-install-recommends \
     linux-image-generic-hwe-20.04 \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-FROM hwe-22-lts AS amd64-ubuntu
-FROM hwe-22-lts AS amd64-ubuntu-22-lts
-FROM hwe-20-lts AS amd64-ubuntu-20-lts
+FROM hwe-22.04 AS amd64-ubuntu-23.04
```

Comment:
22.04 and then 23.04? Typo?

Reply:
This specific "step" `hwe-22.04` is similar both for 23.04 and 22.04 and this way we just avoid introducing an intermediate "common" step. I was also confused the first 3 times I saw this but then I got used to it :D.

Reply:
We put the version in the step names but keep in mind, it's only the BASE_IMAGE that defines what OS we are building on. The rest are steps to configure it and although most of the time, each version has it's own steps, sometimes they are similar. I guess we could avoid naming them with a specific version. Something like `hwe-22.04-and-23.04` would have been more appropriate but you see how this looks.

Reply:
ok, I get it, still a bit confusing IMO